semgrep_output_v1.atd

```diff
@@ -1039,29 +1040,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
+    engine_config: engine_configuration;
 }
-type scan_response_bis = {
-    (* the scan id *)
-    id: string;
-    (* since 1.37.0 *)
-    ?enabled_products: product list;
+
+(* meta info about the scan *)
+type scan_info = {
+    id: string; (* the scan id *)
+    deployment_id: int;
+    deployment_name: string;
+}
+
+(* config specific to the scan, eg *)
+type scan_configuration = {
+    enabled_products: product list;
+    rules: raw_json; (* can we type this better *)
```

Comment:
actually here we're sending a string (that should be interpreted as JSON, but this is still a string)

Reply:
not in the new version of the endpoint